### PR TITLE
Fixed missing use statement and php doc in Queues/Service

### PR DESCRIPTION
**Pull request by konradpodgorski**
I believe this does not require explaining :)

ps there are other issues in this class but I will make separate PRs once I get it running

### CODE REVIEW

File: lib/OpenCloud/Queues/Service.php [hunk 1]
```diff
@@ -117,8 +118,11 @@ public function getClientId()
     /**
      * Create a new Queue.
      *
-     * @param $name Name of the new queue
+     * @param string $name Name of the new queue
+     *
      * @return Queue
+     * 
+     * @throws InvalidArgumentError
```
Comment (GrahamCampbell) [inline lib/OpenCloud/Queues/Service.php]: You need a slash before this.

Comment (GrahamCampbell) [inline lib/OpenCloud/Queues/Service.php]: nvm...

**Comment by jamiehannaford** [comment]
@konradpodgorski Cool! Thanks for submitting this. The linter seems to be complaining about the extra line; to fix, run:

```bash
./vendor/bin/php-cs-fixer fix --level psr2 lib/OpenCloud/Queues/Service.php
```

**Comment by konradpodgorski** [comment]
@jamiehannaford hi. I think it complained because of order, not extra line (which in my opinion is more readable like this)

I ran php-cs-fixer but it didn't change anything

we will see in about minute

**Comment by konradpodgorski** [comment]
@jamiehannaford ok, failed again

Running ```php-cs-fixer fix --dry-run --diff --level psr2 .``` locally with the same settings as in ```.travis``` does not throw anything

**Comment by jamiehannaford** [comment]
@konradpodgorski hmm, when I run the syntax checker it replaces one of the lines with an almost identical version - so maybe it's an encoding issue? 

```
› ./vendor/bin/php-cs-fixer fix --level psr2 lib/OpenCloud/Queues/Service.php
   1) /Users/jamie/projects/php/v1-sdk/lib/OpenCloud/Queues/Service.php
Fixed all files in 0.082 seconds, 5.250 MB memory used

› git diff
diff --git a/lib/OpenCloud/Queues/Service.php b/lib/OpenCloud/Queues/Service.php
index 27c6eb0..506555b 100644
--- a/lib/OpenCloud/Queues/Service.php
+++ b/lib/OpenCloud/Queues/Service.php
@@ -119,7 +119,7 @@ class Service extends CatalogService implements EventSubscriberInterface
      * Create a new Queue.
      *
      * @param string $name Name of the new queue
-     *
+     *
      * @throws InvalidArgumentError
      *
      * @return Queue
```

if you commit the above diff and re-push, does the build still break?

**Pull request merged**